src/view/com/util/post-embeds/BandwidthEstimateContext.tsx

```diff
@@ -0,0 +1,12 @@
+type BandwidthEstimateContext = {
+  getBandwidthEstimate: () => number | undefined
+  setLastestEstimate: (estimate: number) => void
```

Comment:
```suggestion
  setLatestEstimate: (estimate: number) => void
```
lil typo, also in a few other places

Reply:
lol thanks